tests/specs/superNodeProgramModule.spec.js

```diff
@@ -529,6 +587,46 @@ describe('SuperNode program module tests', () => {
             expect(ctrl.okPressed).toEqual(false);
             done();
         });
+
+        it('successful announce transaction when enrolled account update domain', async (done) => {
+            // Arrange:
+            spyOn(Wallet, 'decrypt').and.returnValue(true);
+            spyOn(Wallet, 'transact');
+
+            spyOn(SuperNodeProgram, 'checkEnrollmentStatus').and.returnValue(Promise.resolve(true));
+            spyOn(SuperNodeProgram, 'checkEnrollmentAddress').and.returnValue(Promise.resolve(true));
+            spyOn(SuperNodeProgram, 'getCodewordHash').and.returnValue(Promise.resolve('40DA0977BE27B03B61FBE461225007E1528FD87F20A92BF8E031CDA59C4592E1'));
+
+            const ctrl = $controller('SuperNodeProgramCtrl', {
+                $scope: $rootScope.$new()
+            });
+
+            spyOn(ctrl, 'prepareTransaction');
+
+            ctrl.formData.mainPublicKey = '';
+
+            ctrl.common = {
+                password: '',
+                privateKey: '3c11dc7bc6d8cda5c75b041975362d87eb7de5aef1316f432a1bbfe3e9f19ffb',
+                isHW: false
+            }
+
+            ctrl.enrollStatus.endpoint = 'old.domain.name';
+
+            ctrl.formData.nodeHost = 'new.domain.name';
+
+            // Act:
+            await ctrl.sendEnroll();
+
+            $timeout.flush();
+
+            // Assert:
```

Comment:
should you check model was updateed and contains new domain name ?

Reply:
IMO nope, because it just announced the transaction to the network, and the transaction is pending to confirm. 
the model won't be updated anything.